specs/derivation.md

```diff
@@ -290,7 +292,7 @@ where:
       margin. (A soft constraint is not a consensus rule — nodes will accept such blocks in the canonical chain but will
       not attempt to build directly on them.)
 - `frame_number` identifies the index of the frame within the channel
-- `frame_data_length` is the length of `frame_data` in bytes
+- `frame_data_length` is the length of `frame_data` in bytes. It is capped to 1,000,000 bytes.
```

Comment:
I feel like this should be a power of two? Why this number?

Reply:
This was proto's suggestion. I'd probably go a tad bigger, but I don't have a strong opinion on what it should be. We probably need to look into cannon's limits here

Reply:
Any limit is fine for now, we're not hitting it for now. Doesn't have to be a power of 2. Feel free to change it to a power of 2 though (eth2 specs do it everywhere, except the 1MB network gossip limit, haha).